team/_posts/2017-04-09-sutherland-benjelene.md

```diff
@@ -13,12 +13,12 @@ alumni: false
 cv:
 email: bsuth005@fiu.edu
 github:
-orcid:
+orcid:https://orcid.org/0000-0003-1721-9554
```

Comment:
```suggestion
orcid: 0000-0003-1721-9554
```
You don't need the full URL, plus there should be a space after the field and before the value.